docs/manager/graphql-reference/v2-schema.graphql

```diff
@@ -136,11 +254,191 @@ type ImportArtifactPayload {
   artifact: Artifact!
 }
 
+"""A custom scalar for JSON strings using orjson"""
+scalar JSONString
+
+type MOJOServiceConfig {
+  config: JSONString!
+}
+
+type ModelDeployment implements Node {
+  """The Globally Unique ID of this object"""
+  id: ID!
+  name: String!
+  endpointUrl: String
+  preferredDomainName: String
+  status: DeploymentStatus!
+  openToPublic: Boolean!
+  tags: [String!]!
+  revision: ModelRevision
+  revisionHistory: ModelRevisionConnection!
+  scale: Scale!
+  replicaManagement: ReplicaManagement!
+  deploymentStrategy: DeploymentStrategy!
+  createdUser: UserNode!
+  resourceGroup: ScalingGroupNode!
+  accessTokens: [EndpointToken!]!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
+
+type ModelDeploymentConnection {
+  """Pagination data for this connection"""
+  pageInfo: PageInfo!
+
+  """Contains the nodes in this connection"""
+  edges: [ModelDeploymentEdge!]!
+  count: Int!
+}
+
+"""An edge in a connection."""
+type ModelDeploymentEdge {
+  """A cursor for use in pagination"""
+  cursor: String!
+
+  """The item at the end of the edge"""
+  node: ModelDeployment!
+}
+
+type ModelReplica implements Node {
+  """The Globally Unique ID of this object"""
+  id: ID!
+  name: String!
+  status: ReplicaStatus!
+  revision: ModelRevision!
+  routings: [RoutingNode!]!
+}
+
+type ModelReplicaConnection {
+  """Pagination data for this connection"""
+  pageInfo: PageInfo!
+
+  """Contains the nodes in this connection"""
+  edges: [ModelReplicaEdge!]!
+  count: Int!
+}
+
+"""An edge in a connection."""
+type ModelReplicaEdge {
+  """A cursor for use in pagination"""
+  cursor: String!
+
+  """The item at the end of the edge"""
+  node: ModelReplica!
+}
+
+type ModelRevision implements Node {
+  """The Globally Unique ID of this object"""
+  id: ID!
+  name: String!
+  clusterConfig: ClusterConfig!
+  resourceConfig: ResourceConfig!
+  modelRuntimeConfig: ModelRuntimeConfig!
+  modelVfolderConfig: ModelVFolderConfig!
+  mounts: [Mount!]!
```

Comment:
Let's remove it for now. It seems that the mount configuration in model deployment is sufficient.